docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-dynamic-subprocess-API-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='case-management-dynamic-subprocess-API-proc']
+= Creating a dynamic subprocess using the REST API
+
+When creating a dynamic subprocess, only optional data is provided; there are no special parameters as there are when creating dynamic tasks.
+
+Use the following procedure to create a dynamic subprocess for the IT Orders sample project available in {CENTRAL} using the Swagger REST API tool. The same endpoint can be used for REST API without Swagger.
+
+.Prerequisite 
+You are logged in to {CENTRAL} and an IT Orders case instance has been started using the Showcase application. For more information about using Showcase, see 
+ifeval::["{context}" == "case-management-design"]
+xref:case-management-showcase-application-con-case-management-design[_Case management Showcase application_].
+endif::[]
+
+
+.Procedure 
+. In a web browser, open the following URL:
++
+`http://localhost:8080/kie-server/docs`.
+. Open the list of available endpoints under *Case instances :: Case Management*.
+. Click click the following `POST` method endpoint to open the details: 
++
+`/server/containers/{id}/cases/instances/{caseId}/processes/{pId}`
++
+. Click *Try it out* to complete the following parameters:
++
```

Comment:
Consider a better way to present this information. Perhaps a table with a Parameter, Value, and Description heading? Consider this for all similar cases.